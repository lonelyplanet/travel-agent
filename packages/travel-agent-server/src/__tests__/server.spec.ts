import TravelAgentServer from "../classes/server";

describe("TravelAgentServer", () => {
  it("should work", () => {

  });
})
