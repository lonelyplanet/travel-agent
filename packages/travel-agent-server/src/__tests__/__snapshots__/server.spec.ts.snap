// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TravelAgentServer addModules should add the controller routes to the server's routes 1`] = `
Array [
  Object {
    "controller": Object {
      "default": [MockFunction],
      "name": "TestController",
      "routes": Object {
        "GET /show-stuff": "show",
      },
    },
    "handler": "show",
    "method": "get",
    "middleware": Array [],
    "routes": Object {
      "GET /show-stuff": "show",
    },
    "url": "/show-stuff",
  },
  Object {
    "controller": Object {
      "default": [MockFunction],
      "name": "AnotherController",
      "routes": Object {
        "GET /data": "fetchData",
        "POST /data": "saveData",
      },
    },
    "handler": "fetchData",
    "method": "get",
    "middleware": Array [],
    "routes": Object {
      "GET /data": "fetchData",
      "POST /data": "saveData",
    },
    "url": "/data",
  },
  Object {
    "controller": Object {
      "default": [MockFunction],
      "name": "AnotherController",
      "routes": Object {
        "GET /data": "fetchData",
        "POST /data": "saveData",
      },
    },
    "handler": "saveData",
    "method": "post",
    "middleware": Array [],
    "routes": Object {
      "GET /data": "fetchData",
      "POST /data": "saveData",
    },
    "url": "/data",
  },
]
`;
